packages/mobile/src/store/nativeServices/leaveCommunity/leaveCommunity.saga.ts

```diff
@@ -9,11 +9,8 @@ import { ScreenNames } from '../../../../src/const/ScreenNames.enum'
 
 export function* leaveCommunitySaga(): Generator {
   console.log('Leaving community')
-
   // Restart backend
   yield* putResolve(app.actions.closeServices())
-
-  yield takeLeading(initActions.canceledRootTask.type, clearReduxStore)
```

Comment:
We never want to run `takeLeading` or `takeEvery` more than once because it will result in clearReduxStore being run multiple times. I moved this into the `rootSaga` instead.